src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/DeviceCulture/DeviceCulture.ios.macos.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using Foundation;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class DeviceCulture
+	{
+		private static partial IReadOnlyList<string> GetPreferredCultureStrings() =>
```

Comment:
Why is this a list?

Reply:
Short answer: Because `NSLocale.PreferredLanguages` is a list
Long answer: Because user (device) can have several preferred languages, and if app doesn't support first one, it would be better to use second one instead of just using default (I think you mentioned it somewhere as well).

Reply:
This is wrong. 
You must use the languages from the info.plist file. The first from the list will be that is selected in the app settings. I have provided you with the code aboveюб

Reply:
Wait, this API is meant to get device's language, not application's

Reply:
Please read the articles about iOS localization. We should not use device localization

Reply:
This API is not about localization specifically. It's about getting current language of the device itself (irrespective of the language of the app). And then it can be used for any purposes.

If it's bad practice to use device language for localization on iOS, that users just won't use this API for localization purposes on iOS (platform check is pretty easy in Xamarin).

PS: Just curious, why shouldn't iOS users use device language?